### PR TITLE
Feature/responsive design

### PR DESCRIPTION
**Pull request by lauraguerra1**
Description
<br>
- Tags are laid out in one horizontally scrollable row 
- Recipe grid changes column amounts depending on screen width 
- Page is responsive to cell phones, tablets, and desktop

Testing Notes
<br>
- Open chrome dev tools and click on the mobile and tablet button. 
- Play around with the page at different screen widths

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -88,6 +97,22 @@ tagArea.addEventListener("click", function(event) {
   };
 });
 
+leftArrow.addEventListener('click', () => {
+  tagArea.scrollBy({
+    top: 0,
+    left: -200,
+    behavior: "smooth",
+  })
+})
```
Comment (taranveersingh93) [inline src/scripts.js:114]: lovely

File: src/scripts.js [hunk 2]
```diff
@@ -100,7 +125,8 @@ recipeGrid.addEventListener("mouseover", (event) => {
   }
 });
 
-closeRecipeButton.addEventListener("click", closeRecipe);
+closeRecipeButtons.forEach(btn => btn.addEventListener("click", closeRecipe));
+
```
Comment (taranveersingh93) [inline src/scripts.js:137]: why the refactoring?

**Review by taranveersingh93 — CHANGES_REQUESTED** [review]

File: src/scripts.js (hunk 2 above)
Comment (lauraguerra1) [reply src/scripts.js:137]: I added a new close recipe button when the window is smaller, so they both get the event listener here

**Review by taranveersingh93 — APPROVED** [review]

**Comment by mbenfowler** [comment]
[[wow] Media query enhancement](https://trello.com/c/Q75ZRlLR/58-wow-media-query-enhancement)

**Comment by mbenfowler** [comment]
[[responsive design] change tags to one row with overflow x](https://trello.com/c/wyw05Nmh/59-responsive-design-change-tags-to-one-row-with-overflow-x)

File: src/images/arrow.png
Comment (mbenfowler) [inline src/images/arrow.png:1]: Is this image being used?

File: src/domUpdates.js [hunk 3]
```diff
@@ -429,5 +419,5 @@ export {
   renderActiveTag,
   displayTaggedRecipes,
   renderRecipesOfInterest,
-  enableScrollPitchText
+  enableScrollPitchText,
```
Comment (mbenfowler) [inline src/domUpdates.js]: i guess we don't need this comma?

File: src/styles.css [hunk 4]
```diff
@@ -660,24 +665,100 @@ input[type=checkbox] {
...
+   }
+
+   .bookmark-img {
+    height: 70%;
+   }
 
+   .individual-recipe-image img {
+    display: none;
+   }
+}
```
Comment (mbenfowler) [inline src/styles.css]: newline!

File: src/scripts.js (hunk 1 above)
Comment (mbenfowler) [reply src/scripts.js:114]: `behavior: badass`

**Review by mbenfowler — CHANGES_REQUESTED** [review]
Just a couple minor things

File: src/images/arrow.png
Comment (lauraguerra1) [reply src/images/arrow.png:1]: good call, deleted now

File: src/domUpdates.js (hunk 3 above)
Comment (lauraguerra1) [reply src/domUpdates.js]: fixed in merge conflict resolution, done now

File: src/styles.css (hunk 4 above)
Comment (lauraguerra1) [reply src/styles.css]: done!

**Review by mbenfowler — APPROVED** [review]

**Pull request merged**